docs/deploy-seperate-pods.md

```diff
@@ -12,10 +12,10 @@ This configuration has several advantages:
 - Contour is run as Deployment and Envoy as a Daemonset
 - Envoy runs on host networking
 - Envoy runs on ports 80 & 443
-- The following certificates must be present as Secrets in the `heptio-contour` namespace:
-    - `cacert`: must contain a `CAcert.pem` key that contains a CA certificate that signs the other certificates.
-    - `contourcerts`: must contain `contourcert.pem` and `contourkey.pem` keys that contain a certificate and key for Contour. The certificate must be valid for the name `contour` either via CN or SAN.
-    - `envoycerts`: must contain `envoycert.pem` and `envoykey.pem` keys that contain a certificate and key for Envoy.
+- The following certificates must be present as Secrets in the `heptio-contour` namespace for the example YAMLs to apply:
```

Comment:
these restrictions come from the values we've used in `examples/ds-hostnet-split`. Is it useful to point that out? I'm thinking

> Our example split deployment, `examples/ds-hostnet-split` expects the following: